webview_panels/src/modules/healthCheck/ProjectHealthChecker.tsx

```diff
@@ -277,10 +264,25 @@ const ProjectHealthcheckInput = ({
     }
   }, []);
 
+  const getConfigs = useCallback(async () => {
+    setIsConfigLoading(true);
+    const result = (await executeRequestInSync("getInsightConfigs", {})) as {
```

Comment:
this method needs error handling. If api throws any error, this will fail silently